### PR TITLE
fix(unity) clarify offline default value

### PR DESCRIPTION
**Pull request by bruno-garcia**
Resolves: https://github.com/getsentry/sentry-docs/issues/5508

### CODE REVIEW

File: src/platforms/common/configuration/options.mdx [hunk 1]
```diff
@@ -540,9 +540,27 @@ Configures a separate proxy for outgoing HTTPS requests. This value might not be
...
+
+</ConfigKey>
+
+<ConfigKey name="cache-directory-path" supported={["dotnet.xamarin"]}>
+
+The default value is `Application.persistentDataPath`. You can disable offline caching by setting it to `null`.
+
+</ConfigKey>
+
+The default value is `Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData));`
```
Comment (bruno-garcia) [inline src/platforms/common/configuration/options.mdx:563]:
```suggestion
<ConfigKey name="cache-directory-path" supported={["dotnet.xamarin"]}>

The default value is `Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData));`

</ConfigKey>

```

File: src/platforms/common/configuration/options.mdx [hunk 2]
```diff
@@ -540,9 +540,26 @@ Configures a separate proxy for outgoing HTTPS requests. This value might not be
 
 Specifies a local directory used for caching payloads. When this option is enabled (that is, when the directory is set), the Sentry SDK will persist envelopes locally before sending to Sentry. This configuration option is particularly useful if you expect your application to run in environments where internet connectivity is limited.
 
+</ConfigKey>
+
```
Comment (bruno-garcia) [inline src/platforms/common/configuration/options.mdx:544]:
```suggestion
```

File: src/platforms/common/configuration/options.mdx [hunk 3]
```diff
@@ -536,15 +536,35 @@ Configures a separate proxy for outgoing HTTPS requests. This value might not be
...
+<PlatformSection supported={["android"]}>
+
+Default: set to `android.content.Context.getCacheDir()/sentry`.
+
+</PlatformSection>
+
+<PlatformSection supported={["android", "dotnet"] notSupported={["dotnet.xamarin", "dotnet.maui"]}}>
```
Comment (bruno-garcia) [inline src/platforms/common/configuration/options.mdx:551]:
```suggestion
<PlatformSection supported={["android", "dotnet"]} notSupported={["dotnet.xamarin", "dotnet.maui"]}>
```

File: src/platforms/common/configuration/options.mdx [hunk 4]
```diff
@@ -536,15 +536,35 @@ Configures a separate proxy for outgoing HTTPS requests. This value might not be
...
+<PlatformSection supported={["unity"]}>
 
-</ConfigKey>
+The default value is `Application.persistentDataPath`. You can disable offline caching by setting it to `null`.
+
+</PlatformSection>
+
+<PlatformSection supported={["dotnet.xamarin", "dotnet.maui"]}>
+
+The default value is `Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData));`
```
Comment (mattjohnsonpint) [inline src/platforms/common/configuration/options.mdx:565]: It's different for MAUI.

https://github.com/getsentry/sentry-dotnet/blob/f712c72d2b2b941a2b098ec68401d14e7eb3520a/src/Sentry.Maui/Internal/SentryMauiOptionsSetup.cs#L36

I'll send a corrected PR.

Comment (mattjohnsonpint) [reply src/platforms/common/configuration/options.mdx:565]: See #5519

**Pull request merged**